ui/components/app/snaps/snaps-ui-skeleton/snap-ui-skeleton.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FunctionComponent } from 'react';
+import { Skeleton } from '../../../component-library/skeleton';
+import { BorderRadius } from '../../../../helpers/constants/design-system';
+import { mapSnapBorderRadiusToExtensionBorderRadius } from '../snap-ui-renderer/utils';
+
+export type SnapUISkeletonProps = {
+  width?: number | string;
+  height?: number | string;
+  borderRadius?: BorderRadius;
+};
+
+export const SnapUISkeleton: FunctionComponent<SnapUISkeletonProps> = ({
```

Comment:
Can we just use `Skeleton` instead of introducing a new Snap specific component? It doesn't seem like we need any changes to the component

Reply:
For some components we just add the "regular" component to `safe-component-list` and reference that instead of introducing a Snaps wrapper component, for example:
https://github.com/MetaMask/metamask-extension/blob/main/ui/components/app/snaps/snap-ui-renderer/components/spinner.ts#L5

Reply:
Basically removing the need for a `SnapsUI` prefixed component

Reply:
We don't have specific storybook entries for those at the moment 🤷‍♂️